package-lock.json

```diff
@@ -1224,40 +1224,23 @@
       "resolved": "https://registry.npmjs.org/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz",
       "integrity": "sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw=="
     },
-    "@cmsgov/design-system-core": {
-      "version": "3.7.2",
-      "resolved": "https://registry.npmjs.org/@cmsgov/design-system-core/-/design-system-core-3.7.2.tgz",
-      "integrity": "sha512-IyC/EPU5clQ8wxNLM42ZrUyPLuhuttt8iVv/wg4VRWIb9EgFz2cvVsIRUEsgtz20u5YaxcrXBHIGHpwhvAvZAw==",
+    "@cmsgov/design-system": {
```

Comment:
Obviously we need the `cms/design-system` changes to package-lock.json, but I'm not sure if any of the additional updates below in the file should be included. I used node version 12.22.6 when installing all of the packages.

Reply:
Those look fine to me, just minor updates to other packages.